src/tests/unit/tests/DetailsView/components/tab-stops/tab-stops-requirements-table.test.tsx

```diff
@@ -54,28 +67,37 @@ describe('TabStopsRequirementsTable', () => {
     });
 
     test('renders table', () => {
-        const testSubject = shallow(<TabStopsRequirementsTable {...props} />);
-        expect(testSubject.getElement()).toMatchSnapshot();
+        const testSubject = render(<TabStopsRequirementsTable {...props} />);
+        expect(testSubject.asFragment()).toMatchSnapshot();
     });
 
     test('renders requirement column', () => {
-        const testSubject = shallow(<TabStopsRequirementsTable {...props} />);
-        const columns = testSubject.find(DetailsList).props().columns;
-        expect(columns[0].onRender(requirementContentStub)).toMatchSnapshot();
+        render(<TabStopsRequirementsTable {...props} />);
+        const hasColumns = getMockComponentClassPropsForCall(DetailsList).columns;
+        expect(hasColumns[0].onRender(requirementContentStub)).toMatchSnapshot();
+        expect(hasColumns).toMatchSnapshot();
     });
 
     test('renders result column', () => {
-        const testSubject = shallow(<TabStopsRequirementsTable {...props} />);
-        const columns = testSubject.find(DetailsList).props().columns;
-        const tabStopsChoiceGroup = columns[1].onRender(requirementsList[0]);
+        const testSubject = render(<TabStopsRequirementsTable {...props} />);
+
+        const tabStopsChoiceGroup = testSubject.container.querySelectorAll('.ms-ChoiceFieldGroup');
```

Comment:
This can stay the same as before.
```suggestion
        const columns = getMockComponentClassPropsForCall(DetailsList).columns;
        const tabStopsChoiceGroup = columns[1].onRender(requirementsList[0]);
```

Reply:
Hi
Made changes as per suggestions, please re-review.